### PR TITLE
ADEN-5717 Adds PAL AN placements

### PR DESCRIPTION
**Pull request by leshniak**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js [hunk 1]
```diff
@@ -7,24 +7,36 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', [
...
 	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements',
-		placementsMap = instantGlobals.wgAdDriverAppNexusBidderPlacementsConfig;
+		placementsMap = instantGlobals.wgAdDriverAppNexusBidderPlacementsConfig,
+		palPlacementsMap = {
+			oasis: {
+				atf: '3143028',
+				btf: '3143029',
+				hivi: '3142398',
+				other: '3140173'
+			}
```
Comment (dianafa) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js]: I'm wondering if we could go either one way or the other, that is move all placement ids to the code (like is for other providers) OR keep them all in one place (wf right now)

Comment (dianafa) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js]: per conversation with @nefretete we decided to move all AN placements into code :) Don't care about wg variable set on dev env.

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js [hunk 2]
```diff
@@ -1,19 +1,71 @@
...
+		},
+		palPlacementsMap = {
+			oasis: {
+				atf: '3143028',
+				btf: '3143029',
+				hivi: '3142398',
+				other: '3140173'
+			},
+			recovery: {
+				'atf': '11823778',
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js]: Can we keep it consistent? (without '')

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js [hunk 3]
```diff
@@ -1,19 +1,71 @@
...
+					'hivi': '9412991'
+				},
+				'recovery': {
+					'atf': '11823778',
+					'btf': '11823724',
+					'hivi': '11823799'
+				}
+			}
+		},
+		palPlacementsMap = {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js]: Can we move it to `oasis` object? So we'll have `ent`, `gaming`, `life`, `other`, `recovery`, `pal`

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js [hunk 4]
```diff
@@ -1,19 +1,71 @@
...
+			oasis: {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js]: I believe those IDs are wrong. Please check below:
atf: 11977073
btf: 11977096
hivi: 11977016
other: 11969927

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/f7d56314ea8ddb46345ea2792ad9316c978c1609/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/10b2ec63b8af9b5240498236a6dcf94cdc6cb7c1/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/94b1572e7c256d72d3e7aeaacbc6dacbef4f9e22/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**